cmake/custom/api.cmake

```diff
@@ -5,14 +5,14 @@
 #
 # Variables defined::
 #
-#   ENABLE_Fortran_API
+#   TEST_Fortran_API
 #
 # autocmake.yml configuration::
 #
-#   docopt: "--fbindings=<ENABLE_Fortran_API> Enable compilation of Fortran 90 API bindings <ON/OFF> [default: ON]."
-#   define: "'-DENABLE_Fortran_API={0}'.format(arguments['--fbindings'])"
+#   docopt: "--fbindings=<TEST_Fortran_API> Enable compilation of Fortran 90 API bindings <ON/OFF> [default: ON]."
```

Comment:
Change to `Test the Fortran 90 bindings for the API.`?

Reply:
Done :)